### PR TITLE
Update Slovak translation

### PR DESCRIPTION
**Pull request by kubalav**

### CODE REVIEW

File: Plugins/Flow.Launcher.Plugin.WebSearch/plugin.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "Name": "Web Searches",
   "Description": "Provide the web search ability",
   "Author": "qianlifeng",
-  "Version": "1.3.6",
+  "Version": "1.3.1",
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.WebSearch/plugin.json:29]: Typo here, should be 7

File: Flow.Launcher/Languages/sk.xaml [hunk 2]
```diff
@@ -122,7 +122,6 @@
     <system:String x:Key="newActionKeywordsCannotBeEmpty">Nová skratka pre akciu nemôže byť prázdna</system:String>
     <system:String x:Key="newActionKeywordsHasBeenAssigned">Nová skratka pre akciu bola priradená pre iný plugin, prosím, zvoľte inú skratku</system:String>
     <system:String x:Key="success">Úspešné</system:String>
-	<system:String x:Key="completedSuccessfully">Úspešne dokončené</system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/sk.xaml:125]: This shouldn't be deleted right.

File: Flow.Launcher/Languages/sk.xaml [hunk 3]
```diff
@@ -107,7 +108,6 @@
         alebo prejdite na https://github.com/Flow-Launcher/Flow.Launcher/releases pre manuálne stiahnutie aktualizácie.
     </system:String>
     <system:String x:Key="releaseNotes">Poznámky k vydaniu</system:String>
-	<system:String x:Key="documentation">Tipy na používanie:</system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/sk.xaml:110]: This shouldn't be deleted right

File: Plugins/Flow.Launcher.Plugin.WebSearch/Languages/sk.xaml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:system="clr-namespace:System;assembly=mscorlib">
+
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.WebSearch/Languages/sk.xaml:4]: No need the spacing here right?

**Review by jjw24 — CHANGES_REQUESTED** [review]

File: Plugins/Flow.Launcher.Plugin.WebSearch/Languages/sk.xaml (hunk 4 above)
Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.WebSearch/Languages/sk.xaml:4]:
```suggestion
```

File: Plugins/Flow.Launcher.Plugin.WebSearch/plugin.json (hunk 1 above)
Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.WebSearch/plugin.json:29]:
```suggestion
  "Version": "1.3.7",
```

**Comment by jjw24** [comment]
👍 👍

**Review by jjw24 — APPROVED** [review]

**Pull request merged**